src/components/Home.js

```diff
@@ -0,0 +1,200 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import * as Ruth from '../assets/images/ruth.jpg';
+import * as Hos from '../assets/images/hos.jpg';
+import * as Chim from '../assets/images/face1.jpg';
+
+const Home = () => (
+  <div>
+    <div className="text-center home-banner">
+      <div className="container">
+        <div className="row">
+          <div className="py-5 col-md-12">
+            <h1 className="display-4 text-white">Authors Haven</h1>
+            <h3 className="text-white">
+              Big Ideas
+            </h3>
+            <Link to="Login" className="btn mx-1 btn-secondary">Login</Link>
+            <Link to="SignUp" className="btn btn-primary mx-1">Signup</Link>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div className="pt-5 bg-light">
+      <div className="container">
+        <div className="row text-center">
+          <div className="col-md-6">
+            <h4 className="text-primary">Featured Categories</h4>
+            <ul className="list-group">
+              <Link to="#technology">
+                <li className="list-group-item d-flex justify-content-between align-items-center">
+                  Technology
+                  <span className="badge badge-pill badge-secondary">312</span>
+                </li>
+              </Link>
+              <Link to="#technology">
+                <li
+                  className="list-group-item d-flex justify-content-between align-items-center"
+                >
+                  Lifestyle
+                  <span className="badge badge-pill badge-secondary">299</span>
+                </li>
+              </Link>
+              <Link to="#science">
+                <li
+                  className="list-group-item d-flex justify-content-between align-items-center"
+                >
+                  Science
+                  <span className="badge badge-pill badge-secondary">259</span>
+                </li>
+              </Link>
+              <Link to="#finance">
+                <li
+                  className="list-group-item d-flex justify-content-between align-items-center"
+                >
+                  Finance
+                  <span className="badge badge-pill badge-secondary">258</span>
+                </li>
+              </Link>
+              <Link to="#politics">
```

Comment:
`<li
                  className="list-group-item d-flex justify-content-between align-items-center"
                >
                  Lifestyle
                  <span className="badge badge-pill badge-secondary">299</span>
                </li>`

When a component is too big, consider putting the repeated parts such the one above in a loop for rendering.